docs/source/users/install_instructions/externals.rst

```diff
@@ -109,6 +109,13 @@ RALFit is available to use as fitting software.
 
 Instructions on how to build the python interface are at `<https://ralfit.readthedocs.io/projects/Python/en/latest/install.html>`_
 
+SScanSS-2
```

Comment:
To test this PR, I cloned **SScanSS-2** and then ran `pip install .` in that **SScanSS-2** directory. I also ran `ruff check .` in the fitbennchmarking folder. I saw a lot of errors because sscanss is formatted differently. 
<img width="603" alt="image" src="https://github.com/user-attachments/assets/b4398ec8-160d-4cc3-9e9d-baa180789567" />

Maybe something like this needs to be added to pyproject.toml.

<img width="226" alt="image" src="https://github.com/user-attachments/assets/cbf69412-825d-43bc-b93d-867135811cdb" />

Reply:
Did you clone SScanSS-2 into the fitbenchmarking directory? I get this in the src dir:
```
(fitbenchmarking) alister@SCLT344:~/fitbenchmarking$ cd fitbenchmarking
(fitbenchmarking) alister@SCLT344:~/fitbenchmarking/fitbenchmarking$ ruff check .
All checks passed!
```

Reply:
I usually run the ruff check in the main fitbenchmarking directory. I don't cd into fitbenchmarking. Because it also checks conftest.py is in the main directory and the docs folder. But this can work as well. We can cd into fitbenchmarking and run the ruff checks and if there are any issues with the other files it will be caught by the pre commit bot.

Reply:
I get the same thing in the root of the repo:
```
(fitbenchmarking) alister@SCLT344:~/fitbenchmarking$ pwd
/home/alister/fitbenchmarking
(fitbenchmarking) alister@SCLT344:~/fitbenchmarking$ ruff check .
All checks passed!
```

From your screenshot it looks like SScanSS-2 is in the fitbenchmarking root folder, but since it's not in the repo I don't think we need to add it to the pyproject.toml